frontend/src/components/dashboard/create/email/EmailTemplate.tsx

```diff
@@ -280,9 +280,13 @@ const EmailTemplate = ({
               </h4>
               <p>
                 <label htmlFor="logo">
-                  Your agency’s logo will appear in the header of the email
+                  Your agency’s logo will appear in the header of the email.
                 </label>
               </p>
+              <p>
+                <b>Note:</b> Logos may not be visible in government email
+                inboxes.
+              </p>
```

Comment:
Actually not sure if it's a concern that the `<label>` tag is only in the first para